src/avram/callbacks/callbacks.cr

```diff
@@ -74,7 +74,7 @@ module Avram::Callbacks
         ERR
       end
     %}
-    def after_run(%object)
+    def after_run(%object : T) : Nil forall T
```

Comment:
This might just be `%object : T` and not need the forall... I'll have to double check this.